src/pages/components/variants/inputCheckbox/DefaultInputCheckbox.tsx

```diff
@@ -1,13 +1,23 @@
 import { InputCheckbox } from '@boston-scientific/anatomy-react';
+import { Fieldset } from '@boston-scientific/anatomy-react';
 import Example from 'shared/components/Example';
 
 const DefaultInputCheckbox = (): JSX.Element => {
   return (
-    <Example>
-      <div className="bsds-form-control">
-        <InputCheckbox label="Checkbox" />
-      </div>
-    </Example>
+    <>
+      <Example>
+        <div className="bsds-form-control">
+          <InputCheckbox label="Checkbox" />
+        </div>
+      </Example>
+      <Example>
+        <Fieldset legend="Checkbox group">
```

Comment:
We set the checkbox group's legend text to "Legend" in SB and inconsistent across docs site fieldsets. Let's chat about what we want it to be so we can standardize.